src/download.ts

```diff
@@ -77,16 +79,44 @@ export type ResolveHnsResponse = {
 };
 
 export const defaultDownloadOptions = {
-  ...defaultOptions("/"),
+  ...defaultBaseOptions,
+  endpointDownload: "/",
```

Comment:
why do you have to explicitly add these?

Reply:
the SkyDB options are a combination of registry, download and upload options. This means if someone set the generic `endpointPath` option he would set it for all operations in SkyDB. I needed to make method-specific endpoint options.